src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -54,11 +54,19 @@ public class SendReceiveTest : ClientWebSocketTestBase
             }
         }
 
-        [ActiveIssue(21102, TargetFrameworkMonikers.Uap)]
         [OuterLoop] // TODO: Issue #11345
         [ConditionalTheory(nameof(WebSocketsSupported)), MemberData(nameof(EchoServers))]
         public async Task SendReceive_PartialMessageBeforeCompleteMessageArrives_Success(Uri server)
         {
+            if (PlatformDetection.IsUap && !PlatformDetection.IsWindows10InsiderPreviewBuild16215OrGreater)
```

Comment:
Instead of this code, you could just change the ConditionalTheory to put the checking in that.

Using ConditionalFact / ConditionTheory is preferred if possible.